### DISCUSSION TITLE
Unable to clone Helper script repo

### DISCUSSION THREAD

**Discussion by jbeardon** [Guilouz/Creality-Helper-Script-Wiki#445] [Issues]
Hi Guys.
I've just had to replace the mainboard in one of my K1 Max printers. I've performed the specified factory reset and cannot clone the repo.
Ran through the bits re not checking for SSL and setting the date/time and still no joy. I've forced Google DNS servers via DHCP and have also used my own pihole DNS - same problem.

Screenshot of latest attempt (there have been several):

![image](https://github.com/Guilouz/Creality-K1-Series/assets/14904316/380ca697-3d15-4314-a606-b687287011d2)

Any help would be greatly appreciated 😄 
Thanks
John

**Comment by Guilouz** [comment]
Looks like a network issue. Make sure github is not blocked in your country or in your router.

**Reply by jbeardon** [reply]
That's the weird thing - I'm connected to the same wifi/router while typing this but don't seem to be getting any DNS resolution on the printer, no matter what DNS servers I force via DHCP. 
I'll re-boot everything just to be sure. I did change the IP reservation for the MAC of the new mainboard so it may have caused the router to have a fit.
Thanks for replying so quickly 👍

**Reply by jbeardon** [reply]
I'm an idiot @Guilouz 
Back in September last year, I blocked the IPs of my Maxes from connecting to the internet because they were spamming baidu.com. As soon as the reserved IP kicked in, the block was applied.
Apologies for wasting your time.
Cheers
John

**Reply by Guilouz** [reply]
No problem.
